src/Components/RoomForm/RoomForm.js

```diff
@@ -39,47 +39,45 @@ function RoomForm() {
     })
   }
 
-
-
-    return (
-      <section className="RoomForm">
-        <form onSubmit={ (e) => {e.preventDefault()} }>
-          <label htmlFor='room-input'>Roomname:
-            <input
-              id='room-input'
-              type='text'
-              placeholder='Name'
-              value={ roomname }
-              onChange={(e) => { setRoomname(e.target.value) }}
-            />
-          </label>
-          <h5>Room Code: { roomID }</h5>
-          <section>
-            {createRules()}
-          </section>
-          <label>
-            <input id='rule-input'
-              type='text'
-              placeholder='Rule'
-              value={ potentialRule }
-              onChange={ (e) => { setRule(e.target.value) } }
-              onKeyUp={ (e) => { handleEnter(e) } }
-            />
-          <button type='button' onClick={(e) => { addRule(e) }}>+</button>
-          </label>
-          <Link to='/'>
-            <button type='button'>
-              Cancel
-            </button>
-          </Link>
-          <Link to='/'>
-            <button type='button'>
-              Create Room
-            </button>
-          </Link>
-        </form>
-      </section>
-    );
+  return (
+    <section className="RoomForm">
+      <form onSubmit={ (e) => {e.preventDefault()} }>
+        <label htmlFor='room-input'>Roomname:
+          <input
+            id='room-input'
+            type='text'
+            placeholder='Name'
+            value={ roomname }
+            onChange={(e) => { setRoomname(e.target.value) }}
+          />
+        </label>
+        <h5>Room Code: { roomID }</h5>
+        <section>
+          {createRules()}
+        </section>
+        <label>
+          <input id='rule-input'
+            type='text'
+            placeholder='Rule'
+            value={ potentialRule }
+            onChange={ (e) => { setRule(e.target.value) } }
+            onKeyUp={ (e) => { handleEnter(e) } }
+          />
```

Comment:
Good jobs with the adding rule functionality

Reply:
learning slowly but surely!